src/main.ts

```diff
@@ -14,6 +14,7 @@ export default class FolderIndexPlugin extends Plugin {
 	folderNodeModule: FolderNoteModule;
 	eventManager: EventEmitter
 	oldGraphSetting = false
+	includeFileContent = false;
```

Comment:
See command below

Reply:
I'm not following. Which command? 
I've taken the line out now, but that may not be what you meant.

Reply:
I shouldn't annotate stuff white tired. I meant comment, I'm sorry xD